src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/FileUploader.vue

```diff
@@ -154,14 +172,19 @@ export default defineComponent({
           let fileErrors: string[] = [];
 
           if (file.size > this.maxFileSizeInBytes) {
-            fileErrors.push(`The selected file exceeds the maximum allowed size. Upload a file that is ${this.maxFileSizeInMB}MB or smaller.`);
+            fileErrors.push(`This file is too big. Only files ${this.maxFileSizeInMB}MB or smaller are accepted.`);
           }
           if (file.type !== "application/pdf") {
-            fileErrors.push("The selected file type is not supported. Upload a file with the format of PDF.");
+            fileErrors.push("This type of file is not accepted. File types accepted: PDF");
           }
           if (this.selectedFiles.some((f: FileItem) => f.file.name === file.name)) {
-            fileErrors.push("A file with the same name already exists.");
+            fileErrors.push("This file with this name has already been uploaded. ");
           }
+          if (this.selectedFiles.length + files.length > this.maxNumberOfFiles) {
+            this.showErrorBanner = true;
+            this.errorBannerMessage = `You can only upload ${this.maxNumberOfFiles} files. You need to remove files before you can continue.`;
+          }
+
```

Comment:
Refer to line 41 where we use a hidden input component to prevent the user from progressing. 

![image](https://github.com/user-attachments/assets/190e895b-1773-4272-8c67-97d1b25dde1d)

I think this is a cleaner solution than having the parent check for the errors to display a specific alert message.

Reply:
addressed this one